Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <title>ContentEditable-Disabled</title>
+    <script class='remove'>
+        var respecConfig = {
+            specStatus: "ED",
+            editors: [
+                {
+                    name: "Johannes Wilm",
+                    mailto: "johannes@fiduswriter.org",
+                    company: "Invited Expert"
+                },
+                {
+                    name: "Bo Cupp",
+                    mailto: "pcupp@microsoft.com",
+                    company: "Microsoft"
+                },
+                {
+                    name: "Grisha Lyukshin",
+                    mailto: "glyuk@microsoft.com",
+                    company: "Microsoft"
+                }
+            ],
+            github: "https://github.com/w3c/editing",
+            shortName: "contenteditable-interface",
+            wg: "Web Applications Working Group",
+            wgURI: "https://www.w3.org/2019/webapps/",
+            wgPatentURI: "https://www.w3.org/2004/01/pp-impl/114929/status",
+            license: "w3c-software-doc"
+        };
+    </script>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c-common' class='remove'></script>
+</head>
+
+<body>
+    <section id='abstract'>
+        <h2>Abstract</h2>
+        <p>
+            This specification defines an API that indicates to user agents which editing-related commands are applicable to contenteditable regions of a document.  User agents SHOULD take this information into account when presenting any editing-related UI to ensure its applicability.
+        </p>
+        <p>
+            This specification purposefully does not prescribe the means by which a user agent will indicate to the user that some commands are not applicable in a given contenteditable region.
+        </p>
+    </section>
+    <section id='sotd'>
+      <p></p>
+    </section>
+    <section id='conformance'>
+      <p></p>
+    </section>
+    <section id="introduction">
+        <h2>Introduction</h2>
+        <p>
+            In contenteditable regions of documents, some user agents provide contextual user interface (UI) elements to help accelerate common editing operations.  The UI is meant to enhance the editing experience, but when authors create customized editors that aren’t compatible with user agent UI, it leads to confused users.
+        </p>
+        <p>
+            WYSIWYG markdown editors provide a canonical example. While markdown editors typically support bold or italicized text, they often don't support font color or font family customization. System or browser UI elements like those shown for iOS or the Mac Touch Bar below don't have any indication as to which formatting commands are applicable to a particular contenteditable element and should be displayed, and which ones should be disabled or omitted.
+        </p>
+        <figure>
```

Comment:
I don't think we should have this in the spec either... this could change in the future, and it's not appropriate to call out specific implementations in specification. Maybe move all this to a Wiki page here on GitHub.